react/I18n/translation.jsx

```diff
@@ -27,8 +27,12 @@ export const initTranslation = (
 
   // Load context locales
   if (context) {
-    const dict = dictRequire(lang, context)
-    _polyglot.extend(dict)
+    try {
+      const dict = dictRequire(lang, context)
+      _polyglot.extend(dict)
+    } catch (e) {
+      console.warn(`The context phrases for "${lang}" can't be loaded`)
```

Comment:
Sorry for the late comment but what about a more explicit message, containing the `context` information ?

I suggest :
> ```js
`The context ${context} cannot be loaded for lang ${lang}
```